DBMS module/flow_sequence_diag.md

```diff
@@ -1,6 +1,6 @@
 # Data Flow Diagram
 
-<img width="700" height="535" src="/images/flow-sequensce.png"> 
+<img width="650" height="700" src="/images/flow-sequensce.png" alt="Flow sequence diagram" border= "7" style= "display: block; margin-left: auto; margin-right: auto; ">
```

Comment:
@96RadhikaJadhav I think the border is not yet set....
Is the border visible to you,... 
if yes, then can you attach the screenshot here.